onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_selectors.cc

```diff
@@ -164,11 +192,14 @@ bool ConvSelector::Check(const Graph& graph,
   return dt_bias == ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_INT32;
 }
 
-void ConvSelector::UpdateBuilder(NodesToOptimizeBuilder& builder) const {
-  builder.input_nodes.resize(3);  // add nullptr for bias if missing
+void ConvSelector::UpdateBuilder(NodesToOptimizeIndexesBuilder& builder) const {
+  if (builder.input_nodes.size() == 2) {
+    // add EmptyNodeIndex for bias if missing
+    builder.input_nodes.push_back(NodesToOptimize::EmptyNodeIndex);
```

Comment:
this is perfectly fine, just want to note that the original code could also be adapted to:
`builder.input_nodes.resize(3, NodesToOptimize::EmptyNodeIndex);`